patterns/footer-social.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Title: Centered Footer with social links
+ * Slug: twentytwentyfive/footer-social
+ * Categories: footer
+ * Block Types: core/template-part/footer
+ * Description: Footer with centered site title and social links
+ */
+?>
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|50","left":"var:preset|spacing|50","right":"var:preset|spacing|50"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="padding-top:var(--wp--preset--spacing--60);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50)">
+	<!-- wp:site-title {"level":0,"textAlign":"center","fontSize":"x-large"} /-->
+	<!-- wp:navigation {"overlayMenu":"never","fontSize":"x-large","layout":{"type":"flex","justifyContent":"center"},"ariaLabel":"<?php esc_attr_e( 'Social', 'twentytwentyfive' ); ?>"} -->
+		<!-- wp:navigation-link {"label":"<?php esc_html_e( 'Facebook', 'twentytwentyfive' ); ?>","url":"#"} /-->
+		<!-- wp:navigation-link {"label":"<?php esc_html_e( 'Instagram', 'twentytwentyfive' ); ?>","url":"#"} /-->
+		<!-- wp:navigation-link {"label":"<?php esc_html_e( 'X', 'twentytwentyfive' ); ?>","url":"#"} /-->
+	<!-- /wp:navigation -->
```

Comment:
I did not use the social icon block because, well, the design does not use icons.